csharp/ql/test/library-tests/csharp9/record.expected

```diff
@@ -0,0 +1,81 @@
+types
+| Record.cs:9:1:15:1 | Person | IEquatable<Person> |
+| Record.cs:17:1:23:1 | Teacher | IEquatable<Teacher> |
+| Record.cs:25:1:30:1 | Student | IEquatable<Student> |
+| Record.cs:32:1:32:57 | Person1 | IEquatable<Person1> |
+| Record.cs:34:1:35:35 | Teacher1 | IEquatable<Teacher1> |
+| Record.cs:37:1:38:35 | Student1 | IEquatable<Student1> |
+| Record.cs:40:1:44:1 | Pet | IEquatable<Pet> |
+| Record.cs:46:1:57:1 | Dog | IEquatable<Dog> |
+members
+| Record.cs:9:1:15:1 | Person | Record.cs:9:15:9:20 | != |
```

Comment:
These methods are extracted from source code, but they have no body, and at the same time they are not `abstract` or `extern`. Does this cause any trouble?